### DISCUSSION TITLE
GitHub Student Developer Pack Benefit

### DISCUSSION THREAD

**Discussion by md-ashraful-islam-ullas** [community/community#137596] [GitHub Education] [answered]
### Hi everyone,

I got approved yesterday. Benefits is not available yet. I hope I'll get it soon. 
Now my question is, **Can I claim multiple offers? Or I'm able to claim only one offer?**

**Comment by davevad93** [comment, marked as answer]
Hi @md-ashraful-islam-ullas ,

You can only claim one offer because the benefits are tied to the account and the educational email linked to it, plus as stated in the [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student) "***you cannot transfer academic discounts from one account to another. If you have more than one account you want to apply the discount to, consider [merging](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts) your personal accounts and [renaming](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/changing-your-github-username) the retained account if desired.***"

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @md-ashraful-islam-ullas was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

For further reference you can look at the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
